checkout_sdk/payments/payments.py

```diff
@@ -84,17 +84,35 @@ class ThreeDSFlowType(str, Enum):
     FRICTIONLESS_DELEGATED = 'frictionless_delegated'
 
 
-class PayPalShippingPreference(str, Enum):
+class ShippingPreference(str, Enum):
     NO_SHIPPING = 'NO_SHIPPING'
     SET_PROVIDED_ADDRESS = 'SET_PROVIDED_ADDRESS'
     GET_FROM_FILE = 'GET_FROM_FILE'
 
 
-class PayPalUserAction(str, Enum):
+class UserAction(str, Enum):
     PAY_NOW = 'PAY_NOW'
     CONTINUE = 'CONTINUE'
 
 
+class MerchantInitiatedReason(str, Enum):
+    DELAYED_CHARGE = 'Delayed_charge'
+    RESUBMISSION = 'Resubmission'
+    NO_SHOW = 'No_show'
+    REAUTHORIZATION = 'Reauthorization'
+
+
+class TerminalType(str, Enum):
```

Comment:
If this is added here, then we should remove the reference in payment_apm_four.py
Also for `OsType`